ofl/yarndings20/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,7 @@
+<p>The Soft Type Collection is designed for knitters to chart out their typographic projects. Yarndings is an ornamental typeface inspired by a mix of typographic ornamentation, traditional Fair Isle & Nordic knitting patterns, and dingbat fonts from the 1990's.</p>
+
+<p>Each typeface has a “Regular” and “Charted” version, and some include multiple scales so you can fit type on your knits, no matter the project's size</p>
```

Comment:
The link to the charted version is needed (following the same structure for all the Soft Type fonts)